tests/test-client-shape.lua

```diff
@@ -27,8 +39,17 @@ runner.run_steps{
         assert(not surface.load_silently(c.client_shape_clip, false))
         assert(not surface.load_silently(c.shape_clip, false))
 
+        client.get()[1].fullscreen = false
```

Comment:
What does the `fullscreen = false` changes here? Isn't the client already not fullscreen?

Reply:
whoops, leftover from a previous attempt at hitting that `if`.